### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by DeonFranklin**
PR to get eyes on our back-end structure.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -54,3 +54,7 @@ gem "devise", "~> 4.8"
 gem "react-rails", "~> 2.6"
 
 gem "bootstrap", "~> 5.1"
+
+gem 'rack-cors', :require => 'rack/cors'
+
```
Comment (sjproctor) [inline Gemfile:58]: You don't need `cors` for this project as you are not making a request across origin.

File: app/controllers/blogs_controller.rb [hunk 2]
```diff
@@ -1,2 +1,39 @@
 class BlogsController < ApplicationController
+    
+    def index
+        blogs = Blog.all
+        render json: blogs
+    end
+
+    def create 
+        byebug
```
Comment (sjproctor) [inline app/controllers/blogs_controller.rb]: Don't leave `byebugs` in your code!

**Review by sjproctor — APPROVED** [review]
Just a couple notes. Def remove your byebug!

File: app/javascript/components/App.js [hunk 3]
```diff
@@ -19,18 +17,56 @@ export class App extends Component {
...
-  createBlog = (blog) => {
-    console.log(blog);
-  };
+  componentDidMount(){
+    this.readBlog(),
+    this.readShlf()
+  }
+
+  readBlog(){
+    fetch("http://localhost:3000/blogs")
```
Comment (iameddieyayaya) [inline app/javascript/components/App.js:31]: create a config.js file - so you can create const and import it.

It'll make it easier to update later.

File: app/javascript/components/App.js [hunk 4]
```diff
@@ -19,18 +17,56 @@ export class App extends Component {
...
+    .then(response => response.json())
+    .then(blogsArray => this.setState({blogs: blogsArray}))
+    .catch(errors => console.log("Blog read errors:", errors))
+
+  }
+
+  readShlf(){
+    fetch("http://localhost:3000/shlves")
```
Comment (iameddieyayaya) [inline app/javascript/components/App.js:39]: Same ^

**Pull request merged**